### ISSUE TITLE
Incompatibility of keras 2.2.4-tf with KerasImport

### ISSUE THREAD

**Issue by Valahaar** [eclipse/deeplearning4j#7992]
#### Issue Description

Trying to import a keras model with KerasImport.importKerasModelAndWeights() results in a NullPointerException.

This is due to the fact that, in the function mapOptimizer in KerasOptimizerUtils.java, the variable optimizerParameter is holding the learning rate parameter with key "learning_rate" but the code is looking it up with key "lr".

#### Version Information

DL4J: 1.0.0-beta4
Keras: 2.2.4-tf

#### Contributing

The fix should be easy, just change the parameter from "lr" to "learning_rate".

To make it work until it is fixed, running the following commands from python is enough:
```python
import h5py

# keras' model.save
model.save(MODEL_FILENAME)
model_file = h5py.File(MODEL_FILENAME, 'a')

lr_name_update = model_file.attrs['training_config'].decode().replace('learning_rate', 'lr').encode()
model_file.attrs.modify('training_config', lr_name_update)

model_file.close()
```

**Comment by AlexDBlack** [comment]
Thanks for the report and workaround.
Oddly enough I can't find any reference to "learning_rate" at all in the Keras repo:
https://github.com/keras-team/keras/search?q=learning_rate&unscoped_q=learning_rate

But I've added loading from either "lr" or "learning_rate" property. Will be merged to master at some point soonish.
